pkg/utils/kubernetes/utils.go

```diff
@@ -0,0 +1,37 @@
+package kubernetes
+
+import (
+	"errors"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/discovery"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+func CustomResourceDefinitionExists(gvk schema.GroupVersionKind, cfg ...*rest.Config) (bool, error) {
+	var err error
```

Comment:
Why is this declared as a variable?

Reply:
fixed

Reply:
I don't see this being fixed.

Reply:
after changed cfg to non-variadic, I need this declaration here.

Reply:
ok, I see what's going on, and you're right, at least the way things are with cfg today. Let's come back to this after we figure out the cfg input.